nadlogar/problems/forms.py

```diff
@@ -1,59 +1,40 @@
 from django import forms
 from django.core.exceptions import ValidationError
-from django.forms.models import ModelChoiceIterator
-
-from .models import ProblemText
 
 
 def problem_form(content_type, *args, **kwargs):
     Generator = content_type.model_class()
-    example_problem = kwargs.get("instance", Generator())
-    example_data = example_problem.example_data()
-
-    class ProblemTextChoiceIterator(ModelChoiceIterator):
-        def choice(self, obj):
-            return (self.field.prepare_value(obj), obj.render(example_data))
 
     class ProblemForm(forms.ModelForm):
-        instruction = forms.CharField(
-            label="Navodilo",
-            widget=forms.Textarea(attrs={"rows": 5}),
-            required=False,
-            initial=Generator.default_instruction,
-        )
-        solution = forms.CharField(
-            label="Rešitev",
-            widget=forms.Textarea(attrs={"rows": 3}),
-            required=False,
-            initial=Generator.default_solution,
-        )
+        uses_custom_text = forms.BooleanField(initial=False, required=False)
 
         class Meta:
             model = Generator
             exclude = ["content_type", "document"]
 
         def __init__(self, *args, **kwargs):
             super().__init__(*args, **kwargs)
-            self.fields["text"].queryset = content_type.problemtext_set
```

Comment:
Tega dela ne razumem čisto. Ali je to treba spremeniti zaradi izbrisa razreda ProblemText, ali je samo neodpravljena posledica pull requesta #71?

Reply:
Ne, to je po novem, saj smo odstranili atribut `text`, ki se je prej skliceval na razred `ProblemText`. Tega razreda in s tem polja ni več, zato ni treba povedati, kakšne izbire so na voljo zanj.